cds/modules/theme/static/js/main.js

```diff
@@ -64,11 +64,11 @@ app.controller('mainCtrl', function ($scope, $sce, $q, $http, localStorageServic
     });
     return deferred.promise;
   }
-  function onSelect(selected) {
+  function onSelect(selected, preventFormSubmit) {
```

Comment:
Is this extra parma used somewhere or it is just an option for the future if needed to disable it? I mean, obviously not taking into account where we use it below...

Reply:
it's used here https://github.com/CERNDocumentServer/cds-videos/pull/1596/files#diff-f2f2e32a477870cb4d52b76666ce8350R90 :)

Reply:
actually I just added for the below use but could be used in the future if we want to change behaviour.

Reply:
the thing is that we use different searhbars for the search and the home page so this was a bit of a hack to trigger search on suggestion select.